bin/release.py

```
@@ -0,0 +1,294 @@
+    run_command("git pull --rebase origin main", shell=True)
+
+    update_changelog(version)
+    update_files(version)
+
+    run_command("git add CHANGELOG.md", shell=True)
+    run_command("git add src/", shell=True)
+
+    message = f"""🔖({RELEASE_KINDS[kind]}) release version {version}
+
+Update all version files and changelog for {RELEASE_KINDS[kind]} release."""
+    run_command(f"git commit -m '{message}'", shell=True)
```

Ruff (0.14.10)

Function call with `shell=True` parameter identified, security issue

(S604)